docs/reference/query-dsl/distance-feature-query.asciidoc

```diff
@@ -4,81 +4,37 @@
 <titleabbrev>Distance feature</titleabbrev>
 ++++
 
-The `distance_feature` query is a specialized query that only works
-on <<date, `date`>>, <<date_nanos, `date_nanos`>> or <<geo-point,`geo_point`>>
-fields. Its goal is to boost documents' scores based on proximity
-to some given origin. For example, use this query if you want to
-give more weight to documents with dates closer to a certain date,
-or to documents with locations closer to a certain location.
-
-This query is called `distance_feature` query, because it dynamically
-calculates distances between the given origin and documents' field values,
-and use these distances as features to boost the documents' scores.
-
-`distance_feature` query is typically used on its own to find the nearest
-neighbors to a given point, or put in a `should` clause of a
-<<query-dsl-bool-query,`bool`>> query so that its score is added to the score
-of the query.
-
-Compared to using <<query-dsl-function-score-query,`function_score`>> or other
-ways to modify the score, this query has the benefit of being able to
-efficiently skip non-competitive hits when
-<<search-uri-request,`track_total_hits`>> is not set to `true`.
-
-==== Syntax of distance_feature query
-
-`distance_feature` query has the following syntax:
-[source,js]
---------------------------------------------------
-"distance_feature": {
-  "field": <field>,
-  "origin": <origin>,
-  "pivot": <pivot>,
-  "boost" : <boost>
-}
---------------------------------------------------
-// NOTCONSOLE
-
-[horizontal]
-`field`::
-    Required parameter. Defines the name of the field on which to calculate
-    distances. Must be a field of the type `date`, `date_nanos` or `geo_point`,
-    and must be indexed (`"index": true`, which is the default) and has
-    <<doc-values, doc values>> (`"doc_values": true`, which is the default).
-
-`origin`::
-    Required parameter. Defines a point of origin used for calculating
-    distances. Must be a date for date and date_nanos fields,
-    and a geo-point for geo_point fields. Date math (for example `now-1h`) is
-    supported for a date origin.
-
-`pivot`::
-    Required parameter. Defines the distance from origin at which the computed
-    score will equal to a half of the `boost` parameter. Must be
-    a `number+date unit` ("1h", "10d",...) for date and date_nanos fields,
-    and a `number + geo unit` ("1km", "12m",...) for geo fields.
+Boosts the <<query-filter-context, relevance score>> of documents closer to a
+provided `origin` date or point.
```

Comment:
I think  it is important to include some variation of the below paragraph, as to my mind it nicely highlights the main usage for this query:

"For example, use this query if you want to give more weight to documents with dates closer to a certain date, or to documents with locations closer to a certain location."

Reply:
Re-added with 232a365d405.